app/web/components/AppRoute.tsx

```diff
@@ -0,0 +1,115 @@
+import { Container, useTheme } from "@material-ui/core";
+import classNames from "classnames";
+import CircularProgress from "components/CircularProgress";
+import CookieBanner from "components/CookieBanner";
+import ErrorBoundary from "components/ErrorBoundary";
+import Footer from "components/Footer";
+import { useAuthContext } from "features/auth/AuthProvider";
+import { useRouter } from "next/dist/client/router";
```

Comment:
```suggestion
import { useRouter } from "next/router";
```
Shouldn't this work too based on their example in the docs?